doc/release-notes.md

```diff
@@ -31,7 +31,66 @@ Example item
 
 Example text.
 
-0.12.1 Change log
+
+dash-cli: arguments privacy
```

Comment:
-stdin is not available in Dash as of now. It is introduced in Bitcoin PR https://github.com/bitcoin/bitcoin/pull/7550

Reply:
Ups, that paragraph slipped in, just wanted to merge the Linux ARM part :)